src/test/config.ts

```diff
@@ -46,18 +37,18 @@ export const testAccounts = {
     email: 'superuser@test.com',
     password: 'Pa55word11',
```

Comment:
You have a constant called PASSWORD at the top of this class, can you not use 'config.PASSWORD' here for all the users' password.

Reply:
Removed extra constants